src/Microsoft.Azure.EventHubs.ServiceFabricProcessor/Checkpoint.cs

```diff
@@ -0,0 +1,151 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.using System;
+
+using System;
+using System.Collections.Generic;
```

Comment:
Can you move all using into namespace section?

Reply:
Done.